benchmarks/benchmarks/optimize_linprog.py

```diff
@@ -0,0 +1,90 @@
+"""
+Benchmarks for Linear Programming
+"""
+from __future__ import division, print_function, absolute_import
+
+# Import testing parameters
+try:
+    from scipy.optimize import linprog
+    from scipy.linalg import toeplitz
+    from scipy.optimize.tests.test_linprog import lpgen_2d
+    from scipy._lib._numpy_compat import suppress_warnings
+    import numpy as np
+    import os 
+except ImportError:
+    pass
+
+from .common import Benchmark
+
+methods = ["simplex","interior-point"]
```

Comment:
Style: space after comma here. Note that ``pep8`` ignores quite a few errors, we've set it up that way because older code in SciPy is not compliant. A quick check with ``flake8 in ``benchmarks/benchmarks`` gives output like:

    ./optimize_linprog.py:13:14: W291 trailing whitespace
    ./optimize_linprog.py:19:21: E231 missing whitespace after ','
    ./optimize_linprog.py:21:1: E302 expected 2 blank lines, found 0
    ./optimize_linprog.py:25:24: E231 missing whitespace after ','
    ./optimize_linprog.py:30:1: E302 expected 2 blank lines, found 1
    ./optimize_linprog.py:34:11: E231 missing whitespace after ','
    ./optimize_linprog.py:34:13: E231 missing whitespace after ','
    ./optimize_linprog.py:43:58: W291 trailing whitespace
    ./optimize_linprog.py:45:1: W293 blank line contains whitespace
    ./optimize_linprog.py:46:1: E302 expected 2 blank lines, found 1
    ./optimize_linprog.py:49:17: E231 missing whitespace after ','
    ./optimize_linprog.py:49:21: E231 missing whitespace after ','
    ./optimize_linprog.py:50:17: E231 missing whitespace after ','
    ./optimize_linprog.py:50:21: E231 missing whitespace after ','
    ./optimize_linprog.py:50:25: W291 trailing whitespace
    ./optimize_linprog.py:52:28: E231 missing whitespace after ','
    ./optimize_linprog.py:52:32: E231 missing whitespace after ','
    ./optimize_linprog.py:53:1: W293 blank line contains whitespace
    ./optimize_linprog.py:55:15: E231 missing whitespace after ','
    ./optimize_linprog.py:55:22: E231 missing whitespace after ','
    ./optimize_linprog.py:55:42: E231 missing whitespace after ','
    ./optimize_linprog.py:57:1: W293 blank line contains whitespace
    ./optimize_linprog.py:61:22: E251 unexpected spaces around keyword / parameter equals
    ./optimize_linprog.py:61:24: E251 unexpected spaces around keyword / parameter equals
    ./optimize_linprog.py:61:58: W291 trailing whitespace
    ./optimize_linprog.py:62:27: E127 continuation line over-indented for visual indent
    ./optimize_linprog.py:63:1: W293 blank line contains whitespace
    ./optimize_linprog.py:64:1: E302 expected 2 blank lines, found 1
    ./optimize_linprog.py:66:17: W291 trailing whitespace
    ./optimize_linprog.py:69:28: E231 missing whitespace after ','
    ./optimize_linprog.py:70:1: W293 blank line contains whitespace
    ./optimize_linprog.py:79:25: E231 missing whitespace after ','
    ./optimize_linprog.py:80:17: E221 multiple spaces before operator
    ./optimize_linprog.py:81:1: W293 blank line contains whitespace
    ./optimize_linprog.py:83:24: E251 unexpected spaces around keyword / parameter equals
    ./optimize_linprog.py:83:26: E251 unexpected spaces around keyword / parameter equals
    ./optimize_linprog.py:84:17: E128 continuation line under-indented for visual indent
    ./optimize_linprog.py:84:32: W291 trailing whitespace
    ./optimize_linprog.py:85:17: E128 continuation line under-indented for visual indent
    ./optimize_linprog.py:85:32: W291 trailing whitespace
    ./optimize_linprog.py:86:17: E128 continuation line under-indented for visual indent
    ./optimize_linprog.py:86:32: W291 trailing whitespace
    ./optimize_linprog.py:87:17: E128 continuation line under-indented for visual indent
    ./optimize_linprog.py:88:17: E128 continuation line under-indented for visual indent
    ./optimize_linprog.py:89:17: E128 continuation line under-indented for visual indent
    ./optimize_linprog.py:90:36: E231 missing whitespace after ','

So worth using ``flake8`` on your code.